package/Dockerfile.submariner-networkplugin-syncer

```diff
@@ -1,4 +1,4 @@
-FROM fedora:33 
+FROM fedora:32
```

Comment:
Please add a comment in the Dockerfile to point to the issue, and perhaps the upstream reference (so we know where to look if things break before `go-ovn` is ready).